doc/install.rst

```diff
@@ -115,7 +112,7 @@ Configure and compile COLMAP::
     cd colmap
     mkdir build
     cd build
-    cmake .. -GNinja
+    cmake .. -GNinja -DBLA_VENDOR=Intel10_64lp
```

Comment:
Should we comment on this obscure argument? when is it needed and when is it not?

Reply:
Did my best :-)